frontend/awx/access/credentials/components/CredentialMultilineInput.tsx

```diff
@@ -100,10 +126,39 @@ export function CredentialMultilineInput({
         helperText={handleHelperText(field)}
         isRequired={requiredFields.includes(field.id)}
         isReadOnly={handleIsDisabled(field)}
+        isDisabled={watchedFieldValue?.[field.id] === '$encrypted$'}
+        isClearButtonDisabled={
+          watchedFieldValue?.[field.id] === '$encrypted$' || !watchedFieldValue?.[field.id]
+        }
         allowEditingUploadedText={true}
         icon={
           kind !== 'external' ? (
-            <Button icon={<KeyIcon />} variant="control" onClick={handleModalToggle} />
+            isSecret && watchedFieldValue?.[field.id] === '$encrypted$' ? (
+              <div style={{ display: 'grid' }}>
+                <Button
+                  icon={<KeyIcon />}
+                  variant="control"
+                  onClick={handleModalToggle}
+                  isDisabled
+                />
+                <Tooltip content="Replace">
+                  <Button icon={<UndoIcon />} variant="control" onClick={() => clearField()} />
+                </Tooltip>
+              </div>
+            ) : isSecret &&
+              initialValues?.[field.id] &&
+              watchedFieldValue?.[field.id] !== '$encrypted$' &&
+              !watchedFieldValue?.[field.id]?.includes('Value is managed by') &&
```

Comment:
If this is to check whether or not the value is managed by a credential plug in you can check for it's existence in `accumulatedPluginValues` array.